dbt/adapters/synapse/relation_configs/policies.py

```diff
@@ -0,0 +1,24 @@
+from dataclasses import dataclass
+
+from dbt.adapters.base.relation import Policy
+from dbt_common.dataclass_schema import StrEnum
+
+
+class SynapseRelationType(StrEnum):
+    Table = "table"
+    View = "view"
+    CTE = "cte"
+    MaterializedView = "materialized_view"
+
+
+class SynapseIncludePolicy(Policy):
```

Comment:
Make this a `@dataclass`. Removing it broke it for me.

Reply:
```suggestion
@dataclass
class SynapseIncludePolicy(Policy):
```